public/app/features/explore/state/query.ts

```diff
@@ -330,8 +320,8 @@ export const changeQueries = createAsyncThunk<void, ChangeQueriesPayload>(
     }
 
     // if we are removing a query we want to run the remaining ones
-    if (queries.length < queries.length) {
-      dispatch(runQueries(exploreId));
+    if (queries.length < oldQueries.length) {
```

Comment:
OMG 😨

Reply:
This will change the behavior but to something that actually makes sense 👍 I wonder if users got used to the old behavior that now they may be surprised 🤔 Maybe we could add it to the changelog?

Reply:
opened https://github.com/grafana/grafana/pull/69643 for this